src/app/(frontend)/[center]/weather/stations/map/page.tsx

```diff
@@ -34,12 +34,14 @@ type PathArgs = {
 export default async function Page({ params }: Args) {
   const { center } = await params
   return (
-    <div className="pt-24 pb-24">
-      <div className="container mb-16">
-        <div className="prose dark:prose-invert max-w-none" id="nac-widget-container">
-          <h1>Weather Station Map For {center}</h1>
-          <NACWidget center={center} widget={'stations'} id="nac-widget-container" />
+    <div className="py-12">
+      <div className="container flex flex-col gap-4">
+        <div className="prose dark:prose-invert max-w-none">
+          <h1>
+            <span className="uppercase">{center}</span> Weather Station Map
```

Comment:
Maybe a question for Chris, I originally had put in these blurbs just to make it obvious to me during testing that routing was working and content changed. Since the only thing on the page is a map of weather stations, and we have the current nav highlighted when you're on the page, do we want to invest some valuable top-level screen real estate for this header or could we omit it?